src/search.cpp

```diff
@@ -262,7 +262,8 @@ void Search::search(Workload wl) {
             if (target_queues.find(q) == target_queues.end()) {
                 Indiv n_indiv = Indiv(metric_t::CENQ, q);
                 Comp n_wl_spec = Comp(n_indiv, op_t::LE, 0u);
-                TimedSpec n_spec = TimedSpec(n_wl_spec, total_time, total_time);
+                WlSpec* n_wl_spec_ptr = new Comp(n_wl_spec);
```

Comment:
Merge these lines. No need for separate `n_wl_spec` and `n_wl_spec_ptr`

Reply:
[Search handles pointers more elegantly](https://github.com/all-things-networking/fperf/pull/27/commits/00fe78d47212573296f3abb7df4352785a85d2f3)